src/main/java/org/ibp/api/java/impl/middleware/observationunits/ObservationUnitImportRequestValidator.java

```diff
@@ -149,6 +158,17 @@ public BindingResult pruneObservationUnitsInvalidForImport(
 		return this.errors;
 	}
 
+	private boolean containsPlotNo(final List<ObservationLevelRelationship> observationLevelRelationships) {
```

Comment:
Based on Brapi standard. The ObservationLevelRelationship.levelName only accepts the following enum:
"study", "field", "entry", "rep", "block", "sub-block", "plot", "sub-plot", "plant", "pot", "sample". See this link: 
https://app.swaggerhub.com/apis/PlantBreedingAPI/BrAPI-Phenotyping/2.0#/ObservationUnitHierarchyLevel

Adjust the code so that the "plot" maps to "PLOT_NO", "rep" to "REP_NO", "block" to "BLOCK_NO". This applies to POST observationunits and POST search/observationunits.